gdnative-core/src/init.rs

```diff
@@ -0,0 +1,46 @@
+use crate::core_types::GodotString;
+
+pub struct InitCallbackInfo {
+    pub in_editor: bool,
+    pub active_library_path: GodotString,
+    options: *mut crate::sys::godot_gdnative_init_options,
+}
+
+#[allow(dead_code)]
+impl InitCallbackInfo {
+    pub unsafe fn new(options: *mut crate::sys::godot_gdnative_init_options) -> Self {
```

Comment:
This constructor should be hidden.

Reply:
Is that possible? It's called from the macro used in the consumer's project.

Reply:
Yes. Hidden from docs != private at the language level.

Reply:
ah, got it.